node/task.ts

```diff
@@ -1150,42 +1151,35 @@ function _legacyFindFiles_getMatchingItems(
  * @param     path     path to remove
  * @returns   void
  */
-export function rmRF(path: string): void {
-    debug('rm -rf ' + path);
-
-    // get the lstats in order to workaround a bug in shelljs@0.3.0 where symlinks
-    // with missing targets are not handled correctly by "rm('-rf', path)"
-    let lstats: fs.Stats;
+export function rmRF(inputPath: string): void {
+    debug('rm -rf ' + inputPath);
     try {
-        lstats = fs.lstatSync(path);
-    }
-    catch (err) {
-        // if you try to delete a file that doesn't exist, desired result is achieved
-        // other errors are valid
-        if (err.code == 'ENOENT') {
-            return;
+        if (getPlatform() == Platform.Windows) {
+            if (fs.statSync(inputPath).isDirectory()) {
+                childProcess.execSync(`rd /s /q "${inputPath}"`);
+            }
+            else {
+                childProcess.execSync(`del /f /a "${inputPath}"`);
+            }
+        }
+        else {
+            childProcess.execSync('rm -rf ' + inputPath);
         }
-
-        throw new Error(loc('LIB_OperationFailed', 'rmRF', err.message));
     }
-
-    if (lstats.isDirectory()) {
-        debug('removing directory');
-        shell.rm('-rf', path);
-        let errMsg: string = shell.error();
-        if (errMsg) {
-            throw new Error(loc('LIB_OperationFailed', 'rmRF', errMsg));
+    catch (err) {
```

Comment:
Did we lose the return? Hard to read the diff.

Reply:
Yeah, we don't actually want that anymore though, we want to try to get rid of the dead symlink regardless